lib/__tests__/cli.test.js

```diff
@@ -361,6 +366,22 @@ describe('CLI', () => {
 		expect(output).toBe('Custom formatter reports 1 warning(s).');
 	});
 
+	it('output a message when wrong --globby-options provided', async () => {
+		await cli([
+			`--globby-options=stylelint-test-globby-options`,
+			'--config',
+			fixturesPath('default-severity-warning.json'),
+			fixturesPath('empty-block.css'),
+		]);
+
+		expect(process.exitCode).toBe(2);
+		expect(process.stdout.write).toHaveBeenCalledTimes(0);
+		expect(process.stderr.write).toHaveBeenCalledTimes(1);
+		expect(stripAnsi(process.stderr.write.mock.calls[0][0])).toContain(
+			'Invalid option "--globby-options". The value "stylelint-test-globby-options" is not valid JSON object.',
+		);
+	});
```

Comment:
[suggest] Thanks for adding the case. Additionally, could you add more cases to verify the new feature like this?

```js
it('--globby-options', async () => {
	await cli([
		'--globby-options={"dot":true}',
		'--config',
		fixturesPath('config-block-no-empty.json'),
		fixturesPath('globby-options'),
	]);

	expect(process.stdout.write).toHaveBeenCalledTimes(1);
	expect(process.stdout.write).toHaveBeenCalledWith(expect.stringMatching(/block-no-empty/));
});
```

adding the following file tree:

```tree
lib/__tests__/fixtures/globby-options/
└── .dot/
    └── empty-block.css
```

Also, let's add the new cases using the `cli()` function to the last in `cli.test.js`.

Reply:
Getting error, test cases are failing when modifying conditions. Do let me know if i have to modify anything further! [tried many things but couldn't resolve this]
<img width="1079" alt="Screenshot 2022-11-07 at 4 59 59 PM" src="https://user-images.githubusercontent.com/10995431/200299787-1b86ce22-4014-4339-85be-d615fb8cb7fc.png">

Reply:
Umm..., could you please push your commits (you can leave it as failures)? I'd like to look into the changed test code.

Reply:
Thanks for pushing the commit. Can you update `empty-block.css` as follows?

```diff
diff --git a/lib/__tests__/fixtures/globby-options/.dot/empty-block.css b/lib/__tests__/fixtures/globby-options/.dot/empty-block.css
index e69de29bb..077f6dd7c 100644
--- a/lib/__tests__/fixtures/globby-options/.dot/empty-block.css
+++ b/lib/__tests__/fixtures/globby-options/.dot/empty-block.css
@@ -0,0 +1 @@
+a {}
```

The `a {}` CSS code violates the `block-no-empty` rule.

Reply:
my bad.. sure updated!